README_WEEK01.md

```diff
@@ -0,0 +1,79 @@
+
+# Questions
+
+## How many users do we have?
+
+Query:
+```sql
+    select count(user_uuid) 
+    from dbt_jpramos.stg_users;
+```
+Result: `130 users`
+    
+## On average, how many orders do we receive per hour?
+
+Query:
+```sql
+    with orders_in_hour as (
+        select 
+            date_trunc('hour', created_at_utc),
```

Comment:
I think it should be `date_part`. Date trunc would essentially remove the seconds from the timestamp. So when you do the counting, it's counting per day per hour.

Reply:
yeah, that was my initial though but this makes more sense to me... you want to count *all* the hours that occurred since the beginning of time. I think we'd lose information If we aggregated simply by the hour

Reply:
but we still take all the hours when we do date_part. We just didn't take the day part into the account. Generally, it's a good question how it should be done as date_trunc was also my first try

Reply:
Maybe it's easier if we look at a smaller scenario...
Imagine the following scenario of 2 days with 3 hours:
| Day-hour | Amount |
|----------|--------|
| 1-0      | 10     |
| 1-1      | 5      |
| 1-2      | 2      |
| 2-0      | 5      |
| 2-1      | 15     |
| 2-2      | 20     |

with `date_trunc` we'd get an avg of 9.5/h, whereas with `date_part` we'd get 19/2h. Does that make sense?

Reply:
hm yeah. You are right! Business-wise it's def more meaningful. Thanks for the explanation :)